geoclue-providers-hybris.pri

```diff
@@ -33,6 +33,12 @@ session_dbus_service.path = /usr/share/dbus-1/services
 system_dbus_conf.files = com.jollamobile.gps.conf
 system_dbus_conf.path = /etc/dbus-1/system.d
 
+systemd_dbus_service.files = geoclue-providers-hybris.service
+systemd_dbus_service.path = /usr/lib/systemd/user
+
+systemd_dbus_service_symlink.path = .
+systemd_dbus_service_symlink.commands = ln -s geoclue-providers-hybris.service ${INSTALL_ROOT}/usr/lib/systemd/user/dbus-org.freedesktop.Geoclue.Providers.Hybris.service
```

Comment:
Is this about providing alternative name for the service, or something else? And that "dbus-" prefix is == convention?

Reply:
it was done to have the systemd sandboxing features available for the service.